config.yaml

```diff
@@ -1,14 +1,11 @@
-# Copyright 2021 jose
+# Copyright 2021 Canonical Ltd.
 # See LICENSE file for licensing details.
-#
-# TEMPLATE-TODO: change this example to suit your needs.
-# If you don't need a config, you can remove the file entirely.
-# It ties in to the example _on_config_changed handler in src/charm.py
-#
-# Learn more about config at: https://juju.is/docs/sdk/config
 
 options:
-  thing:
-    default: 🎁
-    description: A thing used by the charm.
+  target:
```

Comment:
For the first standalone version of this charm, we won't need this. We will need something like this for the distributed Loki bundle, but I am undecided whether I like enough the configuration approach, rather than generic codebase with different `metadata.yaml` and `config.yaml` files.